tests/conanbuilder/test_remote.py

```diff
@@ -15,3 +16,12 @@ def test_default_values(remote):
     assert remote.priority == 0
     assert remote.force is False
     assert remote.login is False
+
+
```

Comment:
@stefaneicher Apart from no remote test, I would not want to modify my conan remotes while running any test...